x-pack/qa/full-cluster-restart/build.gradle

```diff
@@ -3,12 +3,18 @@ import org.elasticsearch.gradle.test.RestIntegTestTask
 import org.elasticsearch.gradle.Version
 
 import java.nio.charset.StandardCharsets
-import java.util.regex.Matcher
 
 // Apply the java plugin to this project so the sources can be edited in an IDE
-apply plugin: 'elasticsearch.build'
+apply plugin: 'elasticsearch.standalone-test'
+
 unitTest.enabled = false
 
+repositories {
+    maven {
+        url "https://snapshots.elastic.co/maven"
```

Comment:
I'm confused why this is necessary. The build shouldn't be using any snapshots; it should all be local. Additionally, this was using the official release url before, not just snapshots?

Reply:
@mark-vieira pointed out that the standalone-test plugin already adds this repo. You're right that this shouldn't be using a repo, I'll remove it. I am guessing that this was a leftover item from when we didn't do that...